mockvulnerableapis/migrations/0003_sampledata_node_id_alter_sampledata_url_and_more.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 4.2 on 2024-03-21 09:10
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('mockvulnerableapis', '0002_alter_sampledata_url'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='sampledata',
+            name='node_id',
+            field=models.CharField(default='x1', max_length=50),
+        ),
+        migrations.AlterField(
+            model_name='sampledata',
+            name='url',
+            field=models.CharField(max_length=100),
+        ),
```

Comment:
why is url field being modified?

Reply:
can you confirm if post altering url field, does the unique constraint get removed or not on it?

Reply:
The url field is modified since it is no longer unique. (url, node_id) is used as the primary key now. Yes, the unique constraint gets removed from url.